cmd/serv.go

```diff
@@ -75,26 +73,33 @@ var (
 	alphaDashDotPattern = regexp.MustCompile(`[^\w-\.]`)
 )
 
-func fail(userMessage, logMessage string, args ...interface{}) {
+func fail(userMessage, logMessage string, args ...interface{}) error {
+	fmt.Fprintln(os.Stdout, "")
```

Comment:
Why print?

Reply:
Not sure if this is go specific or not but there appears to be a chance to cause a zombie process and failure to read the Exit status if nothing is outputted on stdout.